inginious/frontend/i18n/fr/LC_MESSAGES/messages.po

```diff
@@ -1582,22 +1583,46 @@ msgstr "Éditer"
 msgid "Course Name"
 msgstr ""
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:81
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:79
+#: inginious/frontend/plugins/course_creation/templates/create_course_modal.html:29
+#: inginious/frontend/templates/task.html:133
+msgid "Group"
+msgstr "Groupe"
+
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:85
 msgid "Comments or description"
 msgstr ""
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:82
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:86
 msgid "Put your opinion or tell us about the problem you have encountered "
 msgstr ""
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:83
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:87
 msgid "Describe and justify the course you want to create "
 msgstr ""
 
-#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:89
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:93
 msgid "Send"
 msgstr ""
 
+#: inginious/frontend/plugins/contact_page/pages/templates/contact_page.html:101
+#, fuzzy
```

Comment:
Some messages may have been automatically translated and have incorrect translations. Also, when some translated messages are changed and the i18n files are updated, the messages location may be lost, so a fuzzy tag is put on those messages. Leave those messages blank, and remove the fuzzy label in i18n files.